zulipterminal/model.py

```diff
@@ -1747,6 +1748,18 @@ def _handle_update_display_settings_event(self, event: Event) -> None:
                 view.message_view.log[msg_pos] = msg_w_list[0]
         self.controller.update_screen()
 
+    def _handle_realm_user_event_update(self, event: Event) -> None:
```

Comment:
It was fine without the _update suffix here too - we want to maintain consistency with the similar functions, just as with the other event types in api_types.py.